janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLConfigOptions.java

```diff
@@ -65,6 +65,13 @@
                 "and always use LOCAL_QUORUM instead",
             ConfigOption.Type.MASKABLE, false);
 
+    ConfigOption<Boolean> EXTERNAL_LOCKING = new ConfigOption<>(
+            CQL_NS,
+            "external-locking",
+            "True to use external locking mechanism instead mechanism provided by JanusGraph",
```

Comment:
Just a nitpick basically, but when just reading this, I'm not sure whether it activates some other external locking mechanism directly or whether it just tells JanusGraph to not use its locking mechanism because I already have my own mechanism.

How about something like: _True to prevent JanusGraph from using its own locking mechanisms because an external locking mechanism is used outside of JanusGraph._ ?

I would probably also add a sentence that makes it clear that users have to employ their own locking mechanism in this case or get inconsistent data.

Reply:
I could not agree with you more.